tailwind.config.js

```diff
@@ -18,13 +18,20 @@ module.exports = {
       "h1-xl": ["40px", "46px"],
     },
     screens: {
-      xxs: "280px",
-      xs: "325px",
-      sm: "450px",
-      md: "550px",
-      lg: "768px",
+      // xxs: "280px",
+      // xs: "325px",
+      // sm: "450px",
+      // md: "550px",
+      // lg: "768px",
+      // xl: "992px",
+      // xxl: "1200px",
```

Comment:
Let's delete this outright. We can always use the commit history to see what the old values were.

Reply:
agree with that